src/config.rs

```diff
@@ -20,7 +19,6 @@ pub struct Config {
     pub wallet_secret_key: Option<Secret<String>>,
     pub dispense_amount: u64,
     pub dispense_asset_id: AssetId,
-    pub max_dispenses_per_minute: u64,
```

Comment:
Why is this getting removed? I though we need to rate-limit this?

Reply:
We have a captcha that prevents spamming of requests already. We can handle many requests at the same time with dependent transactions. We will use the collected fee as a source for the faucet, so it means an infinite amount of tokens.